### ISSUE TITLE
[BUG] declare your accurate tslib dependency version

### ISSUE THREAD

**Issue by xialvjun** [antvis/G6#2713]
```sh
(this will be run only when your dependencies or config have changed)
 > node_modules/.pnpm/@antv/g6-core@0.0.14/node_modules/@antv/g6-core/es/graph/graph.js:1:30: error: No matching export for import "__spreadArray"
    1 │ import { __assign, __extends, __spreadArray } from "tslib";
      ╵                               ~~~~~~~~~~~~~

error when starting dev server:
Error: Build failed with 1 error:
node_modules/.pnpm/@antv/g6-core@0.0.14/node_modules/@antv/g6-core/es/graph/graph.js:1:30: error: No matching export for import "__spreadArray"
    at failureErrorWithLog (/Users/didi/work/data-ecosystem-model-v3/node_modules/.pnpm/esbuild@0.8.55/node_modules/esbuild/lib/main.js:1177:15)
    at buildResponseToResult (/Users/didi/work/data-ecosystem-model-v3/node_modules/.pnpm/esbuild@0.8.55/node_modules/esbuild/lib/main.js:913:32)
    at /Users/didi/work/data-ecosystem-model-v3/node_modules/.pnpm/esbuild@0.8.55/node_modules/esbuild/lib/main.js:1008:20
    at /Users/didi/work/data-ecosystem-model-v3/node_modules/.pnpm/esbuild@0.8.55/node_modules/esbuild/lib/main.js:559:9
    at handleIncomingPacket (/Users/didi/work/data-ecosystem-model-v3/node_modules/.pnpm/esbuild@0.8.55/node_modules/esbuild/lib/main.js:648:9)
    at Socket.readFromStdout (/Users/didi/work/data-ecosystem-model-v3/node_modules/.pnpm/esbuild@0.8.55/node_modules/esbuild/lib/main.js:526:7)
    at Socket.emit (events.js:314:20)
    at addChunk (_stream_readable.js:308:12)
    at readableAddChunk (_stream_readable.js:283:9)
    at Socket.Readable.push (_stream_readable.js:222:10)
 ERROR  Command failed with exit code 1.
```

**Comment by Yanyan-Wang** [comment]
update to v4.1.15 pls. related to https://github.com/antvis/G6/issues/2702

**Comment by liho00** [comment]
its not even solved, same issue on v4.3.3

**Comment by Yanyan-Wang** [comment]
You ts compiler does not know the operator '...'. Upgrade your tslib and ts compiler.
